core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,16 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput' &&
```

Comment:
The condition here isn't quite correct, since it is currently possible to cover multiple choices in a single answer group, or duplicate choices between answer groups (which, at some point, we really should disallow, since only the first one will be invoked). 

Instead, I think this check should analyze the existing answer groups and determine whether each of the answer choices is represented within at least one answer group.

Also, I suggest structuring this as:

```
    if (interactionId === 'MultipleChoiceInput') {
      return $scope.getAnswerChoices().some(function() {
        // check if the answer choice is not represented within any answer group
      });
    }

    return false;
```

The main idea here is to pull out the interactionId check, so that it's easy to add more branches for different interaction ids in the future.